feat(aps): add troubleshooting index (#4584)
scaleway/docs-content: pages/apple-silicon/troubleshooting/index.mdx

---
meta:
  title: Apple silicon - Troubleshooting
  description: Find troubleshooting resources for resolving issues with Apple silicon.
content:
  paragraph: Find troubleshooting resources for resolving issues with Apple silicon.
dates:
  posted: 2025-03-10
categories:
  - apple-silicon
---

<ProductHeader
  productName="Apple silicon troubleshooting"
  productLogo="appleSilicon"

>   description="Troubleshoot common issues with Apple silicon on Scaleway. Find solutions to help resolve problems and get started."

Author: ```suggestion
  description="Learn how to fix common issues with Apple silicon on Scaleway. Find solutions to help resolve problems and get started."
```

>   description="Learn how to fix common issues with Apple silicon on Scaleway. Find solutions to help resolve problems and get started."

Reviewer: ```suggestion
  description="Learn how to fix common issues with Apple silicon on Scaleway."
```

/>

## Featured Pages

<Grid>
    <DefaultCard
      title="Apple silicon FAQ"

>       description="General info on Apple silicon"

Author: ```suggestion
      description="General info on Apple silicon."
```

      url="/faq/apple-silicon/"
      label="See more"
    />

>     <DefaultCard
>       title="Troubleshooting SSH connection issues to Mac mini servers
> "
>       description="This page helps you to resolve connection issues to your Mac mini via SSH."
>       url="/apple-silicon/troubleshooting/cant-connect-using-ssh/"
>       label="See more"
>     />
>     <DefaultCard

Reviewer: ```suggestion
    <DefaultCard
      title="Troubleshooting SSH connection issues to Mac mini servers"
      description="This page helps you to resolve connection issues to your Mac mini via SSH."
      url="/apple-silicon/troubleshooting/cant-connect-using-ssh/"
      label="See more"
    />
    <DefaultCard
```

      title="Troubleshooting remote desktop connections to Mac mini servers"

>       description="This page helps you to resolve remote desktop (VNC) connection issues to your Mac mini."

Reviewer: ```suggestion
      description="Learn how to solve remote desktop (VNC) connection issues to your Mac mini."
```

      url="/apple-silicon/troubleshooting/cant-connect-using-vnc/"
      label="See more"
    />
</Grid>
<Grid>
    <DefaultCard

>       title="Troubleshooting account creation for hosted Mac minis"

Reviewer: ```suggestion
      title="Fixing Mac mini account creation errors"
```

>       description="This page helps you to resolve Apple account creation issues from your hosted Mac muni"

Reviewer: ```suggestion
      description="Common Apple account creation issues from your hosted Mac mini and how to fix them."
```

      url="/apple-silicon/troubleshooting/cant-create-apple-account/"
      label="See more"
    />
    <DefaultCard
      title="Troubleshooting deletion issues with Mac minis"

>       description="This page helps you to resolve issues you may encounter when you want to delete your Mac mini."

Reviewer: ```suggestion
      description="Resolve issues you may encounter when trying to delete a Mac mini."
```

      url="/apple-silicon/troubleshooting/cant-delete-mac-mini/"
      label="See more"
    />
    <DefaultCard
      title="Bare Metal Shared Responsibility Model"

>       description="Our shared responsibility model clarifies the division of duties between Scaleway and our users."

Reviewer: ```suggestion
      description="Understand the division of duties between Scaleway and our users."
```
